linux_os/guide/services/obsolete/r_services/service_rlogin_disabled.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,ol7
```

Comment:
@cipherboy actually no. CCE applies to `systemctl disable rsh` not disabling in `xinet.d` as well, so if debian uses xinetd, this rule does not apply to debian.

Reply:
I'm not sure I follow how it could not apply. From the description of this rule:

```
description: |-
    The <tt>rlogin</tt> service, which is available with
    the <tt>rsh-server</tt> package and runs as a service through xinetd or separately
    as a systemd socket, should be disabled.
    If using xinetd, set <tt>disable</tt> to <tt>yes</tt> in <tt>/etc/xinetd.d/rlogin</tt>.
    {{{ describe_socket_disable(socket="rlogin") }}}
```

This describes disabling in `xinetd`. Further, both of the CCEs listed are `@product` restricted and the RHEL6 one included here does not use `systemctl`. 

Newer versions of Debian (>= 8) use systemd so this actually _does_ apply to Debian/Ubuntu/etc. as well:

https://wiki.debian.org/systemd#systemd_-_system_and_service_manager

Reply:
@cipherboy shouldn't use a web page that references xinetd. ;-)

*sigh* I guess at some point we are going to have to fix the rule which is not a priority right now.